### PR TITLE
fix(nasha): remove number limit for now use allocated size

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nasha-refresh` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9893 MANAGER-9899 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I removed the partitions number limit for now use allocated size

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/modules/nasha/src/dashboard/dashboard.routing.js [hunk 1]
```diff
@@ -60,8 +58,20 @@ export default /* @ngInject */ ($stateProvider) => {
...
+        iceberg(`${nashaApiUrl}/partition`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute()
+          .$promise.then(({ data }) => {
+            let partitionAllocatedSize = 0;
+            data.forEach((partition) => {
+              partitionAllocatedSize += partition.size;
+            });
+            return partitionAllocatedSize;
```
Comment (JeremyDec) [inline packages/manager/modules/nasha/src/dashboard/dashboard.routing.js:71]: **niptick**
```suggestion
            return data.reduce((totalSize, partition) => {
              return totalSize += partition.size
            }, 0);
```

You can also replace the return (in suggestion), by shortcuts

Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/dashboard.routing.js:71]: With the suggestion I have the message: "Return statement should not contain assignment.eslint[no-return-assign](https://eslint.org/docs/rules/no-return-assign)"

Or you have to:

```
.$promise.then(({ data }) => {
            return data.reduce((totalSize, partition) => {
              const total = totalSize + partition.size;
              return total;
            }, 0);
          }),
          ```

Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/dashboard.routing.js:71]: As this it's working:

```
   .$promise.then(({ data }) =>
            data.reduce(
              (totalSize, partition) => totalSize + partition.size,
              0,
            ),
          ),
```

Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/dashboard.routing.js]: It's applied

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**